packages/default/src/additional/categorization/index.ts

```diff
@@ -0,0 +1,8 @@
+import { registerStartupRenderer } from '@jsonforms/core';
+import CategorizationRenderer from './CategorizationRenderer';
+import { categorizationTester } from './tester';
+
```

Comment:
Shouldn't we export all the components and the renderer to allow an easy extension?